CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/DateTimeRangeInputPane.java

```diff
@@ -534,13 +552,22 @@ public LocalDate fromString(final String s) {
         final VBox picker = new VBox(dpBox, spinnerBox);
         picker.setStyle("-fx-padding:4; -fx-border-radius:4; -fx-border-color: grey;");
 
-        dp.getEditor().textProperty().addListener(changed);
+        dp.getEditor().textProperty().addListener((final ObservableValue<? extends String> v, final String o, final String n) -> {
+            if (n != null && n.isEmpty()) {
```

Comment:
Would it make any sense to swap this out for `StringUtils.isEmpty(n)`? Assuming you may want to do something similar if `n` is null?

Reply:
I found out that `n` cant be null, so I've replaced the `n != null && n.isEmpty()` with just `StringUtils.isEmpty(n)`